modules/README.md

```diff
@@ -1,3 +1,18 @@
+# NOTE
```

Comment:
-  please have this description in root ./README.md file

-  ./modules/complete-eks-cluster also moved into https://github.com/dasmeta/terraform-aws-eks/tree/main , please remove this submodule and have it in readme